UdpSocket.js

```diff
@@ -112,7 +112,7 @@ UdpSocket.prototype.bind = function(port, address, callback) {
 
 UdpSocket.prototype.close = function (callback=noop) {
   if (this._destroyed) {
-    return process.nextTick(callback)
+    return setImmediate(callback)
```

Comment:
any particular reason?

Reply:
process.nextTick doesn't exist on React Native. This line was generating an error

Reply:
ah, right, i've gotten so used to using rn-nodeify, i've forgotten i'm not in node. Good catch!